src/AppStore.ts

```diff
@@ -35,6 +46,23 @@ export class AppStore {
         }
     }
 
+    @computed get undismissedSiteErrors(){
+        const me =  _.filter(this.siteErrors.slice(), (err)=>!err.dismissed);
+        return me;
+    }
+
+    @computed get isErrorCondition(){
+        return this.undismissedSiteErrors.length > 0;
+    }
+
+    @action
+    public dismissErrors(){
+        this.siteErrors = this.siteErrors.map((err)=>{
```

Comment:
`this.siteErrors.forEach` would be more idiomatic, since you're actually making the changes in place.